Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -630,9 +630,12 @@ fileprivate extension URLSessionTask {
         var result = [("Connection", "keep-alive"),
                       ("User-Agent", userAgentString),
                       ]
+        #if !os(Android)
+        // Crashes on Android https://bugs.swift.org/browse/SR-2588
```

Comment:
I’ll look into it. Do you have any pointers? Is this the sort of thing: https://github.com/mapbox/mapbox-gl-native/wiki/Android-debugging-with-remote-GDB